duckduckgo/macos-browser
# Send error with fetch device failure pixel

Task/Issue URL: https://app.asana.com/0/0/1208738052951188/f

**Description**:

The pixel `sync_refresh_devices_error` didn't include the error parameter due to confusion about how the slightly newer PixelKit types do this. This PR just includes the error.

**Steps to test this PR**:
1. Make sure you have your mac and another device (e.g iPhone) synced.
2. Launch this branch in the debugger
3. On the other device, remove the macOS device that is being debugged from sync
4. On the macOS device, go to the sync screen
5. You should see the following sent Pixel info in the console: `m_mac_debug_sync_refresh_devices_error ["d": "DDGSync.SyncError", "pixelSource": "browser-dmg", "e": "28", "appVersion": "1.114.0"]`

**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
